salesagility/SuiteCRM
# Multi AOP

## Description




ability to join multiple joomla site to suitecrm

## Motivation and Context


## How To Test This


join multiple joomla site to crm, try each functionality in cases module with multiple users...
test for backward compatibility, upgrade an older version of suitecrm and add more joomla sites

at the admin panel, check is multiple validation type correct for us and do we need the validation here?
test somehow the marking as undeleted was success? 
at updating portal, check which joomla url need for us to send because since it's multiple it does not make sense

## Types of changes


### Final checklist
